liquibase-core/src/main/java/liquibase/snapshot/SnapshotControl.java

```diff
@@ -97,8 +105,22 @@ private void setTypes(Set<Class<? extends DatabaseObject>> types, Database datab
         }
     }
 
+    private boolean exclude(Class<? extends DatabaseObject> type) {
```

Comment:
Please move this method together with getter below the addType method, which is currently the only place where it is used

Reply:
Applied changes